app/_components/infinite-scroll/LoadMoreItems.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { Item, fetchData } from '@/app/_components/infinite-scroll/fetchData';
+import { useEffect, useState, useCallback } from 'react';
+import { useInView } from 'react-intersection-observer';
+import Items from './Items';
+import Spinner from '../spinner/Spinner';
+
+const LoadMore = () => {
+  const [items, setItems] = useState<Item[]>([]);
+  const [page, setPage] = useState(5);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isEnded, setIsEnded] = useState(false);
+
+  const { ref, inView } = useInView();
+
+  const delay = async (ms: number) => {
+    await new Promise(resolve => setTimeout(resolve, ms));
+  };
+
+  const loadMoreItems = useCallback(async () => {
+    setIsLoading(true);
+    await delay(777);
+
+    const newProducts = (await fetchData(page)) ?? [];
+
+    if (newProducts.length === 0) setIsEnded(true);
+
+    setItems((prevProducts: Item[]) => [...prevProducts, ...newProducts]);
+    setPage(page + 5);
```

Comment:
여기서도 업데이터 함수를 쓸 수 있을 것 같아요..!
```suggestion
    setPage((prevPage) => prevPage + 5);
```